core/modules/replica/testConfiguration.cc

```diff
@@ -89,6 +92,10 @@ BOOST_AUTO_TEST_CASE(ConfigurationTest) {
         {"worker:worker-A.fs_host",      "host-A"},
         {"worker:worker-A.fs_port",      "52001"},
         {"worker:worker-A.data_dir",     "/data/A"},
+        {"worker:worker-A.db_host",      "host-A"},
+        {"worker:worker-A.db_port",      "53306"},
+        {"worker:worker-A.db_user",      "qsmaster"},
+        {"worker:worker-A.db_password",  "CHANGEME"},
```

Comment:
you're sure you want this on GitHub?

Reply:
That `CHANGEME` suggests this is some mock "password", especially in the in-memory unit test 😸 The production environment should not be using this type of passwords.